docs/3.x/querying-the-reporting-api.md

```diff
@@ -32,22 +29,6 @@ Here is the output of this code:
 {@include escape https://piwik.org/api_rest_call.php}
 ``` 
 
-## Call the Piwik API in PHP
-
-If you want to request data in a PHP script **that is on the same server as Piwik**, you can use this simple technique. This is a more efficient solution as it doesn't require network calls. You directly call the PHP Piwik runtime and get the PHP data structure back.
-
-If you are developing a plugin, you should be using this technique. Please note that including index.php will set the default timezone of the calling code to UTC.
-
-```php
-{@include escape https://raw.github.com/matomo-org/matomo/master/misc/others/api_internal_call.php}
```

Comment:
Guess we could remove the included file then as well?

Reply:
Sounds good to me, I can issue a PR later. It seems to be used in a test but not quite sure why.